osquery/bro/QueryManager.cpp

```diff
@@ -0,0 +1,168 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <iostream>
+#include <list>
+#include <sstream>
+
+#include <osquery/flags.h>
+#include <osquery/logger.h>
+
+#include "osquery/bro/QueryManager.h"
+
+namespace osquery {
+
+std::string QueryManager::addOneTimeQueryEntry(const SubscriptionRequest& qr) {
+  const auto queryID = std::to_string(nextUID_++);
+  auto status = addQueryEntry(queryID, qr, "ONETIME");
+  if (status.ok()) {
+    return queryID;
+  } else {
+    LOG(WARNING) << status.getMessage();
+  }
+  return "-1";
```

Comment:
This is strange, will an empty string not convey the same message or is there a reason you can't do that?